Resources/Prototypes/Roles/Jobs/Science/research_director.yml

```diff
@@ -29,7 +29,10 @@
     components:
     - type: BibleUser # Nyano - Lets them heal with bibles
     - type: Psionic # Nyano - They start with telepathic chat
-    - type: DispelPower # Nyano - They get the Dispel psionic power on spawn
+    - type: InnatePsionicPowers
+      powersToAdd:
+        - DispelPower
+        - MetapsionicPower
```

Comment:
Death says one or both of these should be on the suit and not innate, unlike Mantis.

Reply:
No. We can have multiple powers now, and the Mystagogue having both innate is part of the new feature set.